README.md

```diff
@@ -1,47 +1,73 @@
-# OOP-School-Library`
-Here in this project I have created a class `Person` with the following:
-- Instance vars: `@id`, `@name`, and `@age`.
-- Constructor with `name`, `age`, and `parent_permission` as parameter. name and `parent_permission` are optional and have default values of `"Unknown"` and `true`.
-- Getters for `@id`, `@name`, and `@age`.
-- Setters for `@name` and `@age`.
-- Private method `is_of_age?` that returns `true` if `@age` is greater or equal to 18 and false otherwise.
+# School Library
+In this project, I will be building your school library app. In this initial step, I will implement the classes to represent students parents and teachers.
 
-![](./assets/schoolLibrary.PNG)
-![](./assets//DecoratorClass.PNG)
-![](./assets/entery1.PNG)
-![](./assets/entery2.PNG)
-![](./assets/entery3.PNG)
 
-## Project requirements
-The project requirements can be found [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/school_library_people.md)
+## Technologies Used
 
-## Built with
-Programming language: `Ruby`
-Frameworks: `NaN`
-Technology: `N/A`
-Testing tool: [Rubocup](https://rubocop.org/)
+* Language (Ruby)
 
-## Installation
-- Git clone `https://github.com/oluyaratosin123/OOP-School-Library.git`
-- run `cd OOP-School-Library` in your command line
+## Getting Started
 
-## Authors
+To get a local copy up and running follow these simple example steps.
 
-👤 **Author1**
+## Prerequisites
+* A text editor(preferably Visual Studio Code)
 
+## Install
+* Ruby
+
+## Using it Locally
+
+* Clone the project
+
+* Clone the repo using the command below
+
+```
+git clone git@github.com:charlesgobina/school-library.git
+```
+
+* Navigate to the directory in which you cloned the project and use the command below
+
+```
+cd school-library
+```
+
+* Start the program using the command below
+```bash
+open terminal
+run the command $irb (interactive ruby shell)
+run the program functions
+```
+
+OR 
+
+* In the program directory, run the command below to execute individaual files
+
+`ruby person.rb`, `ruby student.rb`, `ruby teacher.rb`
+
```

Comment:
- [ ] Please kindly update your `readme file` with explicit steps to clone and run your app locally as this will be of help to non technical users. For example: 
- `git clone git@github.com:oluyaratosin123/OOP-School-Library.git`
- `cd OOP-School-Library`
- `ruby main.rb`.....